code/__HELPERS/type2type.dm

```diff
@@ -59,7 +61,35 @@
 		if("SOUTHWEST")
 			return SOUTHWEST
 		else
-	return
+			CRASH("Tried to convert text [direction] to dir, but received unrecognised direction.")
+
+//Turns a direction into a nautical direction
+/proc/dir2nautical(direction)
+	switch(direction)
+		if(NORTH)
+			return "fore"
+		if(SOUTH)
+			return "aft"
+		if(EAST)
+			return "starboard"
+		if(WEST)
+			return "port"
+		else
+			CRASH("Tried to convert dir [direction] to nautical, but received unrecognised direction.")
+
+//Turns a nautical direction into a direction
+/proc/nautical2dir(direction)
+	switch(uppertext(direction))
+		if("FORE")
+			return NORTH
+		if("AFT")
+			return SOUTH
+		if("STARBOARD")
+			return EAST
+		if("PORT")
+			return WEST
```

Comment:
would be cool if FORE/AFT/STARBOARD/PORT were defines like the cardinal directions "are"

Reply:
That can be done!